src/why_sponsor.njk

```diff
@@ -0,0 +1,110 @@
+---
+title: Why Sponsor? - PyCon India 2025
+description: Sponsor PyCon India 2025 and support the Python community
+layout: base.njk
+---
+
+{% from "components/button.njk" import button %}
+{% from "components/window.njk" import window %}
+<div class="w-full h-auto bg-pycon-blue -mb- px-[10%] pt-[100px] md:pt-[200px]">
+    <div class="flex flex-col md:flex-row">
+        <div class="pb-8 md:pb-16 w-full lg:w-[70%]">
+            <div class="black-han-sans-regular font-normal text-3xl md:text-5xl md:text-center text-[#FFFFFF] text-center lg:text-left">
+                Why Sponsor?
+            </div>
+            <div class="text-md lg:text-xl pt-12 text-[#FFFFFF] lg:pr-20 md:pb-64">
+                <p>PyCon India 2025 is back in Bengaluru, Karnataka, allowing our community to come together to network, learn, share
+                    ideas, and create new relationships and partnerships.</p>
+                <p class="pt-4">Your sponsorship helps us make PyCon India accessible to developers from diverse backgrounds and
+                    supports the growth of the Python community in India.</p>
+                <p class="pt-4">We invite you to be a part of this vibrant community and showcase your brand at the largest Python
+                    conference in India. Join us in making PyCon India 2025 a success and help shape the future of Python in India!</p>
+                <p class="pt-4">As a sponsor, you will have the opportunity to connect with a diverse audience of Python enthusiasts,
+                    developers, and industry leaders. Your support will not only enhance your brand visibility but also contribute to the
+                    growth and sustainability of the Python community in India. We offer a range of sponsorship packages to suit your needs,
+                    from title sponsorship to community support. Each package is designed to provide maximum exposure and engagement
+                    opportunities for your brand.</p>
+            </div>
```

Comment:
We are missing the callout to checkout our sponsorship brochure. We should add that.